src/ophys_etl/modules/suite2p_registration/suite2p_utils.py

```diff
@@ -0,0 +1,116 @@
+import h5py
+import numpy as np
+from suite2p.registration.register import pick_initial_reference
+from suite2p.registration.rigid import (apply_masks, compute_masks, phasecorr,
+                                        phasecorr_reference, shift_frame)
+
+
+def load_initial_frames(file_path: str,
+                        h5py_key: str,
+                        nimg_init: int) -> np.ndarray:
+    """Load a subset of frames from the hdf5 data specified by file_path.
+
+    Parameters
+    ----------
+    file_path : str
+        Location of the raw 2Photon, HDF5 data to load.
+    h5py_key : str
+        Name of the dataset to load from the HDF5 file.
+    nimg_init : int
+        Number of frames to load from the input HDF5 data.
+
+    Returns
+    -------
+    frames : array-like, (nimg_init, Ly, Lx)
+        Selected frames from the input raw data linearly spaced in index of the
+        time axis. If nimg_init > nframes, a number of frames equal to nframes
+        is returned.
+    """
+    with h5py.File(file_path, "r") as hdf5_file:
+        # Get a set of linear spaced frames to load from disk.
+        tot_frames = hdf5_file[h5py_key].shape[0]
+        requested_frames = np.linspace(0,
```

Comment:
It's nonobvious why the `[:-1]`. I think it is clearer if you pass `endpoint=False`